### PR TITLE
Move out sdtype validations from multi-column transformers

### LINKED ISSUE #778: Move out sdtype validations from multi-column transformers
### Problem Description

Currently, all the sdtype validations for multi-column transformers are defined in their classes.
We want to move these validations out of the classes into a separate file so that they can be used without accessing the transformers. 

### Expected behavior

- Create a `_validations.py` file inside `rdt/transformers`
- Move out all the sdtype validations for the multi-column transformer inside this file
- Make the changes so the multi-column transformer usage stays the same.

### PR DESCRIPTION
**Pull request by R-Palazzo**
CU-86azy5u67
Resolve #778

### CODE REVIEW

**Comment by sdv-team** [comment]
Task linked: [CU-86azy5u67 RDT - Move out sdtype validations from multi-column transformers #778](https://app.clickup.com/t/86azy5u67)

File: rdt/transformers/_validations.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        This method aims to call all other sdtype validation method in the class.
+
+        Args:
+            columns_to_sdtypes (dict):
+                Mapping of column names to sdtypes.
+        """
+        for attr_name in dir(cls):
+            attr = getattr(cls, attr_name)
+            if callable(attr) and attr_name.startswith('_validate_'):
+                attr(columns_to_sdtypes)
```
Comment (amontanez24) [inline rdt/transformers/_validations.py:43]: this is a little tricky. I think it's more transparent to just have the child class implement `validate_sdtypes` and call what it needs to

**Review by amontanez24 — COMMENTED** [review]
One comment but otherwise looks good! I just want to look at the corresponding SDV PR before appproving

File: rdt/transformers/_validations.py (hunk 1 above)
Comment (R-Palazzo) [reply rdt/transformers/_validations.py]: Alright, done in [aac20d5](https://github.com/sdv-dev/RDT/pull/779/commits/aac20d553fdcf5c03ae4dc60d1fe37fbfee0be96)

**Review by amontanez24 — COMMENTED** [review]
One minor change is I would rename `_validations` -> `_validators`

**Review by amontanez24 — APPROVED** [review]
LGTM!

**Pull request merged**